### ISSUE TITLE
Server login page doesn't always show keyboard focus

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5557]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.x
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem
- Load the RStudio sign-in page in web browser
- Use tab key to move around

### Describe the problem in detail
- Not showing keyboard focus location in a consistent fashion

### Describe the behavior you expected
- Should be consistent with main UI, where we've started using the focus-visible selector (https://github.com/rstudio/rstudio/pull/5452)

Also note there's an issue in Pro discussing a visual refresh of the login page, so take that into consideration: https://github.com/rstudio/rstudio-pro/issues/1258
